base/setup/reactos/lang/zh-CN.rc

```diff
@@ -129,7 +129,7 @@ END
 
```

Comment:
Uh there's some trivial problems I found **Above** this line but can't choose them due to GitHub limitation:

* Line 58: The access key should be capitalise but that line doesn't.
* The colon is half-width, maybe they can set to full-width?